### DISCUSSION TITLE
Problem with Obsidian after updating the plugin

### DISCUSSION THREAD

**Discussion by LeonelRFF** [logancyang/obsidian-copilot#957] [Q&A]
Hi, excuse me. Is there something I can configure to improve the speed of the plugin? After updating to 2.7.10 my vault takes much longer to start. Before the update, when I opened Obsidian, everything took about 3 seconds. Now, after the update, it takes about 20 seconds.

![Captura de pantalla 2024-12-20 153615](https://github.com/user-attachments/assets/0555de65-95a1-4198-a7c6-869604860400)
![Captura de pantalla 2024-12-20 153958](https://github.com/user-attachments/assets/1646a751-9fb3-42b7-a9ca-f5a342d5412e)

**Comment by logancyang** [comment]
How big is your existing index? Are you on mobile and disabled indexing? That "features may be limited" means your index was not loaded. You can "force re-index" or manually remove the copilot index files in `.obsidian` and index again.

A quick thing to try is to open the dev console and hit cmd + R to refresh, and see if it persists.

**Comment by LeonelRFF** [comment]
The vault is about 6000 files between notes, images and pdfs.
I think I found the problem, I use ollama and in the QA Settings>Embedding Models the nomic-embed-text:latest model the “Enabled” box is checked, I always had it checked because it didn't take too long, but now it is.
